.github/ISSUE_TEMPLATE/04_user_support.yml

```diff
@@ -0,0 +1,27 @@
+name: User Support
+description: Do you need assistance with admiral? If you found a bug, documentation issue or have a feature request please use other forms.
+title: "Guidance and support on how to use admiral"
```

Comment:
I do not think GitHub is the right place to ask for user support. In fact many repos explicitly state that you should not open issues asking how to do certain things. I feel like our Slack channel and (going forward) StackOverflow would be the right place to ask.